src/common/hooks/auth/use-sign-in.ts

```diff
@@ -78,7 +78,7 @@ export function useSignIn() {
 
   const handleSetSeed = useCallback(
     async (value: string) => {
-      const trimmed = value.trim();
+      const trimmed = value;
```

Comment:
This change makes a few other parts of the code different now.

I'd actually recommend keeping this function as-is and having `onPaste` use this, and then create a different method for `onChange` (which is what would be used for someone manually typing the seed phrase)

Reply:
Sure !

Reply:
@aulneau To avoid code duplication, I added an option to trim since the behaviour is really similar